recipes/biorbd-viz/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "biorbd-viz" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/{{ name }}/archive/Release_{{ version }}.tar.gz
+  sha256: 299466974a8f30204730529c722aa78b618df1b7e23663503387633bc47723b6
+  
+build:
+  skip: True  # [py<35]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - biorbd
+    - vtk
+    - pyqt
+    - pyomeca
+    - xorg-x11-server-xvfb-cos6-x86_64  # [linux]
```

Comment:
You need to make a file called `recipes/biorbd-viz/yum_requirements.txt` which contains:
```bash
xorg-x11-server-Xvfb
```

Reply:
Now it seems it wants to load, but there is missing dependencies on qt I think..
```This application failed to start because it could not find or load the Qt platform plugin "xcb"```
I've tried different things in the meta file and in the yum_requirements file, but I run out of ideas now..

Reply:
@chrisburr thx :)
Now it seems it wants to load, but there is missing dependencies on qt I think..
This application failed to start because it could not find or load the Qt platform plugin "xcb"
I've tried different things in the meta file and in the yum_requirements file, but I run out of ideas now..